content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -1,47 +1,66 @@
 
-.. image:: media/multi_sales_team01.png
-    :align: center
+On the creation page, you can set an *Email Alias* for your team. Every message sent to that email
+address will create a lead/opportunity in your pipeline. You can also choose whether you
+accept emails from *Everyone*, *Authenticated Partners*, or *Followers Only*.
 
-Add members to your sales channel
----------------------------------
+You can set up a *Domain* if you want to assign leads/opportunities to your team based on specific
+filters, such as country, language, etc. Set an *Invoicing Target* if your team has specific
+monthly revenue targets they need to achieve.
 
-You can add members to any channel; that way those members will see the
-pipeline structure of the sales channel when opening it. Any
-lead/opportunity assigned to them will link to the sales channel.
-Therefore, you can only be a member of one channel.
+.. image:: multiple_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: Creating a Sales Team
 
-This will ease the process review of the team manager.
+Add members to your Sales Team
+------------------------------
 
-.. image:: media/multi_sales_team02.png
-    :align: center
+You can add team members to a Sales Team by clicking **Add** under the Assignment tab when editing
+the Sales Team's configuration page. Choose a salesperson by selecting them from the dropdown menu
+or creating a new salesperson. To ensure that your salespeople are not overworking, you can set a
+maximum number of leads that they can be assigned in a 30-day period.
 
-If you now filter on this specific channel in your pipeline, you will
-find all of its opportunities.
+.. image:: multiple_sales_teams/add-a-salesperson.png
+   :align: center
+   :alt: Adding a Salesperson
 
-.. image:: media/multi_sales_team03.png
-    :align: center
+You can add someone as a team member or a Team Leader to multiple Sales Teams, allowing them to
+easily check all of the pipelines they need access to.
 
-Sales channel dashboard
-=======================
+Sales Team dashboard
+====================
+
+To view your Sales Team dashboard, go to :menuselection:`CRM --> Configuration --> Sales Teams`. Any
+teams you are a part of will appear as dashboard tiles.
+
+Each tile gives an overview of the Sales Team's open opportunities, quotations, sales orders, and
+expected revenue, as well as a bar graph of new opportunities per week and an invoicing progress
+bar.
+
+.. image:: multiple_sales_teams/sales-team-overview.png
+   :align: center
+   :alt: Sales Team Overview
+
+Clicking on the three dots in the corner of a tile opens a navigational menu that lets you quickly
+view documents or reports, create new quotations or opportunities, pick a color for this team, or
+access its configuration page.
+
+.. image:: multiple_sales_teams/team-overview-three-dot-menu.png
+   :align: center
+   :alt: Three Dot Menu
+
+Clicking on the **Pipeline** button brings you directly to that team's CRM pipeline.
```

Comment:
Improving tense

`Click on the **Pipeline** button to go directly to that team's CRM pipeline.`